infra/dev/postgres/init.sql

```diff
@@ -8,18 +8,67 @@ CREATE DATABASE "test";
 CREATE USER twenty PASSWORD 'twenty';
 ALTER USER twenty CREATEDB;
 
+-- Create role for pg_graphql
+CREATE ROLE anon;
```

Comment:
"anon" is not a clear name. are we forced to use it?

Reply:
Hmm don't think so, I just use the same name as in the pg_graphql example